test/xbgpu/test_engine.py

```diff
@@ -1452,3 +1555,153 @@ def get_stream_status(stream_name: str) -> bool:
 
             await client.request("capture-start", output.name)
             assert get_stream_status(output.name) is True, f"Stream {output.name} is still disabled"
+
+    @pytest.mark.parametrize(
+        "n_ants, n_channels, n_jones_per_batch, heap_accumulation_threshold",
+        [(4, 1024, 262144, [20, 20])],
+    )
+    @pytest.mark.parametrize("n_x_streams_to_stop, n_b_streams_to_stop", [(2, None), (1, 1), (None, 2)])
+    async def test_capture_stop_some_streams(
+        self,
+        monkeypatch: pytest.MonkeyPatch,
+        mock_recv_streams: list[spead2.InprocQueue],
+        mock_send_stream: list[spead2.InprocQueue],
+        client: aiokatcp.Client,
+        n_ants: int,
+        n_channels_per_substream: int,
+        frequency: int,
+        n_samples_between_spectra: int,
+        n_spectra_per_heap: int,
+        heap_accumulation_threshold: list[int],
+        corrprod_outputs: list[XOutput],
+        beam_outputs: list[BOutput],
+        n_x_streams_to_stop: int,
+        n_b_streams_to_stop: int,
+        xbengine: XBEngine,
```

Comment:
Can you file a bug on the ordering dependency that breaks the test if `xbengine` is earlier in the parameter order? pytest doesn't guarantee anything about how order of fixtures in the function signature affects fixture initialisation order, so it's quite possible that this will come back to bite us in a future pytest version.

Reply:
Will do.